BesterLive/AppDelegate.swift

```diff
@@ -14,23 +14,23 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
   var window: UIWindow?
 
 
-  func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) ->Bool{
+  func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject:AnyObject]?) -> Bool {
     // Override point for customization after application launch.
     print("helllllo")
     return true
   }
 
-  func applicationWillResignActive(application:UIApplication){
+  func applicationWillResignActive(application: UIApplication) {
     // Sent when the application is about to move from active to inactive state. This can occur for certain types of temporary interruptions (such as an incoming phone call or SMS message) or when the user quits the application and it begins the transition to the background state.
     // Use this method to pause ongoing tasks, disable timers, and throttle down OpenGL ES frame rates. Games should use this method to pause the game.
   }
 
-  func applicationDidEnterBackground(application:UIApplication){
+  func applicationDidEnterBackground(application: UIApplication) {
     // Use this method to release shared resources, save user data, invalidate timers, and store enough application state information to restore your application to its current state in case it is terminated later.
     // If your application supports background execution, this method is called instead of applicationWillTerminate: when the user quits.
   }
 
-  func applicationWillEnterForeground(application : UIApplication){
+  func applicationWillEnterForeground(application : UIApplication) {
```

Comment:
Reporter: CHECKSTYLE
Rule: ?
Severity: WARN
File: /Users/framgia/.jenkins/jobs/BesterLive/workspace/BesterLive/AppDelegate.swift L33
 

Colons should be next to the identifier when specifying a type.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>